wandb/proto/wandb_settings.proto

```diff
@@ -265,6 +265,7 @@ message Settings {
   google.protobuf.StringValue _tmp_code_dir = 49;
   ListStringValue _unsaved_keys = 51;
   google.protobuf.BoolValue _windows = 52;
+  google.protobuf.BoolValue allow_offline_artifacts = 177;
```

Comment:
could you please move it after: `google.protobuf.BoolValue _show_operation_stats = 176;` and add a docstring explaining this new setting. 

we are slowly trying to document all the settings that were added during the years, so it would be good to not chase our tail here :)